uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -406,13 +406,14 @@ define([
        * @return {Object} attributes
        */
       this._initRequestAttributes = function () {
+        initUserRole.call(this);
```

Comment:
This should not be there, look at what this method is doing. This method is for the leave request attributes, not for the settings of the popup.